packages/components/filters/filters-list/src/filters-list.tsx

```diff
@@ -7,17 +7,20 @@ import FlatButton from '@commercetools-uikit/flat-button';
 import { PlusThinIcon, CloseIcon } from '@commercetools-uikit/icons';
 import SelectInput from '@commercetools-uikit/select-input';
 
-export type TAppliedFilterValue = {
+export type TAppliedFilterValue = { label: ReactNode; value: string };
+
+export type TAppliedFilter = {
   filterKey: string;
-  label: ReactNode[];
+  values: TAppliedFilterValue[];
 };
 
-export type TFiltersConfiguration = {
+export type TFilterConfiguration = {
```

Comment:
I think this configuration API for each filter is working well enough to implement it.